test/unit/codegeneration/PureES6Transformer.js

```diff
@@ -48,4 +48,44 @@ suite('PureES6Transformer.js', function() {
     var transformed = transformer.transform(tree);
     assert.equal(write(transformed), write(expectedTree));
   });
+
+  test('Inline', function() {
+    var options = new Options({
+      get modules() { return "inline"; },
+      get rootModules() { return [{name: null}]; }
+    });
+    var reporter = new ErrorReporter();
+
+    var code = [
+      "import {TestA} from './resources/test_a.js';",
```

Comment:
We use single quotes everywhere... but in this case you can use a template literal.